_test/tests/inc/parser/parser_media.test.php

```diff
@@ -16,21 +16,20 @@ function testVideoOGVExternal() {
         $calls = array (
             array('document_start',array()),
             array('p_open',array()),
-            array('externalmedia',array($file,null,null,null,null,'cache','details',array())),
+            array('externalmedia',array($file,null,null,null,null,'cache','details',array('controls'=>true))),
             array('cdata',array(null)),
             array('p_close',array()),
-            array('document_end',array()),
-            array('videoAtts',array())
+            array('document_end',array())
         );
         $this->assertEquals(array_map('stripbyteindex',$parser_response),$calls);
 
         $Renderer = new Doku_Renderer_xhtml();
-        $url = $Renderer->externalmedia($file,null,null,null,null,'cache','details',array(),true);
+        $url = $Renderer->externalmedia($file,null,null,null,null,'cache','details',array('controls'=>true),true);
         //print_r("url: " . $url);
-        $video = '<video class="media" width="320" height="240" controls="controls">';
-        $this->assertEquals(substr($url,0,66),$video);
+        $video = '<video class="media" width="320" height="auto" controls="controls">';
```

Comment:
Could you explain the reasoning behind the change from fixed values to auto?

Reply:
Sure. Mostly to match the existing behavior for images. You should only need to define one explicit size (width). The video tag will automatically scale the other dimension properly based on the video file. 

In fact, it may make sense for this to match image behavior entirely and not define a default size at all. By default, use the full size of the video itself. 

This is particularly import for playing back webms (as an alternative to gifs) where they may already be small in dimensions and the default sizing doesn't fit at all. Looking up the actual dimensions to override for the embed, from a non-technical end-user, would be a nightmare.

Reply:
I see. Good idea to assume the default size of the movie is fine.